content/blog/2019/10/2019-10-21-thinking-about-jenkins-security.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Thinking About Jenkins Security - DevOps World 2019"
+tags:
+- jenkins world
+- jenkinsworld
+- devopsworld2019
+- security
+- event
+- community
+authors:
+- markewaite
+- wadeck
+- stackscribe
+opengraph:
+  image: /images/security/do-not-run-malicious-code.png
+---
+
+Securing a Jenkins server is a multi-layered effort that needs to consider networks, operating systems, applications, source code, and more.
```

Comment:
This syntax is troublesome and I'm not sure how to fix it -- but an effort is not a sentient being who can consider anything...  "Securing a Jenkins server requires that you consider ..." -- not crazy about that.  "... multilayered effort that involves networks..." might work.  Or break it into two sentences:  "Securing a Jenkins server is a multi-layered effort.  You must consider networks, operating systems..."  I might use a semi-colon rather than a period but I may overuse semi-colons..

Reply:
See b0094d2891a41859cc96e76a71ccc542fba61917 for my attempt at better phrasing